src/features/home/components/Services.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Dimensions, FlatList, ScrollView, StyleSheet, View, ViewStyle } from 'react-native';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { Map } from '../../../components/Map';
+import { Widget } from '../../../components/Widget';
+
+import { COLORS } from '../../../constants/colors';
+import { MOCK_CARS, Service } from '../../../constants/mock';
+
+type ServicesPropsType = {
+    activeCarIndex: number
+}
+
+const width = Dimensions.get('screen').width;
+const widgetColors = [[COLORS.white, COLORS.heartOfIce], [COLORS.white, COLORS.touchOfMint]];
+
+export const Services = (props: ServicesPropsType) => {
+  const { activeCarIndex } = props;
+  const { bottom } = useSafeAreaInsets();
+
+  const mainWidgets = MOCK_CARS[activeCarIndex].services.slice(0, 2);
+  const commonWidgets = MOCK_CARS[activeCarIndex].services.slice(2);
+
+  const mapContainerStyle = { marginBottom: bottom + 75 };
+
+  const renderItem = (item: Service, styles: ViewStyle, colors?: string[]) => (
+    <Widget widget={item} containerStyle={styles} colors={colors} />
+  );
+
+  return(
+    <ScrollView showsVerticalScrollIndicator={false} style={styles.container}>
+      <FlatList
+        horizontal={true}
+        data={mainWidgets}
+        scrollEnabled={false}
+        renderItem={({ item, index }) => renderItem(item, styles.mainWidget, widgetColors[index])}
+        showsHorizontalScrollIndicator={false}
+      />
+      <FlatList
+        horizontal={true}
+        data={commonWidgets}
+        scrollEnabled={false}
+        renderItem={({ item }) => renderItem(item, styles.commonWidget)}
```

Comment:
write just renderItem={renderItem} here (it's the point of renderItem function to get rid of anonymous arrow function)